Documentation/install/upgrade.rst

```diff
@@ -166,6 +166,8 @@ latest ``1.1.y`` release before subsequently upgrading to ``1.2.z``.
 +-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
 | ``1.1.x``             | ``1.2.y``             | Required              | Temporary disruption[2] | Clients must reconnect[1] |
 +-----------------------+-----------------------+-----------------------+-------------------------+---------------------------+
+| ``1.2.x``             | ``1.3.y``             | Required              | Minimal to None         | Clients must reconnect[1] |
```

Comment:
Do we have any guidance around "Minimal to none"? I think it's likely fine to leave it as is, just wondering if there are any known outstanding issues that might cause disruption in this case.

Reply:
It's a bit hard to measure because the only known impact is the iptables flush and reinsert. This is subsecond. Happy about suggestions on exact wording. We are not quite to the point of guaranteed none yet.

Reply:
@joestringer Do you want a change on this or not?

Reply:
I figured that if there's a fairly straightforward thing to communicate, we would do so in a similar fashion to the existing comments. But it sounds like it's pretty involved.

So, no I'm fine with this as-is.